src/io/flutter/view/FlutterView.java

```diff
@@ -631,15 +631,27 @@ public void perform(AnActionEvent event) {
   public static final String SHOW_REPAINT_RAINBOW = "ext.flutter.repaintRainbow";
 
   RepaintRainbowAction(@NotNull FlutterApp app) {
-    super(app, "Show Repaint Rainbow", "Show Repaint Rainbow", FlutterIcons.RepaintRainbow);
+    super(
+      app,
+      "Show Repaint Rainbow",
+      "Hide Repaint Rainbow",
+      "Show Repaint Rainbow",
```

Comment:
nit: make a constructor with two less parameters that lets you avoid repeating yourself
when the names match.

Reply:
Refactored.